<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEffectDialog</class>
 <widget class="QDialog" name="ItemEffectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Effect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="recoverTab">
      <attribute name="title">
       <string>Recover</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
         <item row="3" column="2">
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="gainTPValueSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="gainTPRadioButton">
           <property name="text">
            <string>Gain TP</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>+</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="recoverMPValue1SpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="recoverHPValue2SpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-100000</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="recoverMPValue2SpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>-100000</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="recoverHPValue1SpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="recoverMPRadioButton">
           <property name="text">
            <string>Recover MP</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>+</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="recoverHPRadioButton">
           <property name="text">
            <string>Recover HP</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stateTab">
      <attribute name="title">
       <string>State</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1">
         <item row="2" column="1" colspan="2">
          <widget class="SelectorButton" name="removeStateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="addStateRadioButton">
           <property name="text">
            <string>Add State</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="addStateSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="removeStateRadioButton">
           <property name="text">
            <string>Remove State</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="SelectorButton" name="addStateButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="removeStateSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="buffsTab">
      <attribute name="title">
       <string>Buffs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,1">
         <item row="4" column="1" colspan="2">
          <widget class="AttribComboBox" name="removeBuffComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="AttribComboBox" name="addDebuffComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="addBuffRadioButton">
           <property name="text">
            <string>Add Buff</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="addDebuffLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>  turns</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="AttribComboBox" name="removeDebuffComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="removeDebuffRadioButton">
           <property name="text">
            <string>RemoveDebuff</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="removeBuffRadioButton">
           <property name="text">
            <string>Remove Buff</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="addDebuffRadioButton">
           <property name="text">
            <string>Add Debuff</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="addDebuffSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="addBuffLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>  turns</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="addBuffSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="AttribComboBox" name="addBuffComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherTab">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,1">
         <item row="1" column="1" colspan="2">
          <widget class="AttribComboBox" name="growComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="growRadioButton">
           <property name="text">
            <string>Grow</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="commonEventRadioButton">
           <property name="text">
            <string>Common Event</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="specialEffectRadioButton">
           <property name="text">
            <string>Special Effect</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_32">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>32</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="growSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="learnSkillRadioButton">
           <property name="text">
            <string>Learn Skill</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="SelectorButton" name="learnSkillButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="SelectorButton" name="commonEventButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="specialEffectComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Escape</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AttribComboBox</class>
   <extends>QComboBox</extends>
   <header>attrib_combo_box.hpp</header>
  </customwidget>
  <customwidget>
   <class>SelectorButton</class>
   <extends>QPushButton</extends>
   <header>selector_button.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEffectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEffectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="radioGroup"/>
 </buttongroups>
</ui>
